space-cli/cmd/utils/helm.go

```diff
@@ -13,46 +13,28 @@ import (
 	"helm.sh/helm/v3/pkg/chart"
 	"helm.sh/helm/v3/pkg/chart/loader"
 	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/release"
+
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/model"
 )
 
 // HelmInstall install helm chart
 func HelmInstall(chartReleaseName, chartLocation, downloadURL, namespace string, valuesFileObj map[string]interface{}) (*chart.Chart, error) {
-	settings := cli.New()
-	actionConfig := new(action.Configuration)
-	if err := actionConfig.Init(settings.RESTClientGetter(), settings.Namespace(), os.Getenv("HELM_DRIVER"), log.Printf); err != nil {
+	actionConfig, helmChart, err := createChart(chartLocation, downloadURL)
+	if err != nil {
 		return nil, err
 	}
 
-	var helmChart *chart.Chart
-	var err error
-	if chartLocation == "" {
-		res, err := http.Get(downloadURL)
-		if err != nil {
-			return nil, err
-		}
-		if res.StatusCode != http.StatusOK {
-			return nil, fmt.Errorf("received invalid status code (%s)", res.Status)
-		}
-		helmChart, err = loader.LoadArchive(res.Body)
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		helmChart, err = loader.Load(chartLocation)
-		if err != nil {
-			return nil, err
-		}
-	}
-
 	iCli := action.NewInstall(actionConfig)
 	iCli.ReleaseName = chartReleaseName
-	iCli.CreateNamespace = namespace != ""
+	iCli.CreateNamespace = namespace != model.HelmSpaceCloudNamespace // don't create namespace while setting up space cloud cluster
```

Comment:
Just make this false
```suggestion
	iCli.CreateNamespace = false
```